ui/cypress/e2e/explorer.test.ts

```diff
@@ -735,7 +735,7 @@ describe('DataExplorer', () => {
         cy.getByTestID('time-machine-submit-button').click()
         cy.getByTestID('cog-cell--button').click()
         cy.getByTestID('select-group--option')
-          .contains('Custom')
+          .last()
```

Comment:
Had to make this very small tweak to our existing OSS e2e tests. With the ability to configure the tick marks enabled by the flag `axisTicksGenerator`, there are more customization options. The work-around to keep this test working is to select the last "Custom" button that appears now - which happens to be the one for setting the min/max values on the Y-axis.